cli.py

```diff
@@ -0,0 +1,166 @@
+    )
+    pedersen_hasher_parser.add_argument(
+        "-p", "--personalisation", help="Provide personalisation string", default="test"
+    )
+
+    # keygen subcommand
+    keygen_parser = subparsers.add_parser(
+        "keygen",
+        help="Returns space separated hex-string for a random private/public keypair on BabyJubJub curve",
+    )
+    keygen_parser.add_argument(
+        "-p",
+        "--from_private",
+        help="Provide existing private key as hexstring (64 chars)",
+    )
+
+    # eddsa signature generation subcommand
+    sig_gen_parser = subparsers.add_parser(
+        "sig-gen",
+        help="Returns eddsa signature as space separated hex-string. Private key and message needs to be provided",
+    )
+    sig_gen_parser.add_argument(
+        "private_key", nargs=1, help="Provide public key as hexstring (64chars)"
+    )
+
+    sig_gen_parser.add_argument(
+        "message", nargs=1, help="Provide message as string, will be utf-8 encoded"
+    )
+
+    # eddsa signature verify subcommand
+    sig_verify_parser = subparsers.add_parser(
+        "sig-verify", help="Verifies a eddsa signaure. Returns boolean flag for success"
+    )
+    sig_verify_parser.add_argument(
+        "public_key", nargs=1, help="Provide public key as hexstring (64chars)"
+    )
+    sig_verify_parser.add_argument(
+        "message", nargs=1, help="Provide message as string, will be utf-8 encoded"
+    )
+    sig_verify_parser.add_argument(
+        "signature",
+        nargs=2,
+        help="Provide signaure as space separated hexsting (2x 64 chars)",
+    )
+
+    args = parser.parse_args()
+    subparser_name = args.subparser_name
+
+    if subparser_name == "hash":
+        preimage = bytes.fromhex(args.preimage[0])
+        if len(preimage) != args.size:
+            raise ValueError(
+                "Bad lenght for preimage: {} vs {}".format(len(preimage), args.size)
+            )
+
+        personalisation = args.personalisation.encode("ascii")
+        point = PedersenHasher(personalisation).hash_bytes(preimage)
+        digest = point.compress()
+
+        assert len(digest.hex()) == 32 * 2  # compare to hex string
+        print(digest.hex())
+
+    elif subparser_name == "batch_hasher":
+        personalisation = args.personalisation.encode("ascii")
+        ph = PedersenHasher(personalisation)
+        try:
+            while True:
+                x = input()
+                if x == "exit":
+                    exit(0)
+                preimage = bytes.fromhex(x)
+                if len(preimage) != args.size:
+                    raise ValueError(
+                        "Bad length for preimage: {} vs {}".format(len(preimage), 64)
+                    )
+                point = ph.hash_bytes(preimage)
+                digest = point.compress()
+                assert len(digest.hex()) == 32 * 2  # compare to hex string
+                print(digest.hex())
```

Comment:
If this is not a leftover from debugging, I would recommend adding some clarification text to the `print` function about what the output is.